go-controller/pkg/ovn/loadbalancer/loadbalancer.go

```diff
@@ -194,3 +195,29 @@ func GetWorkerLoadBalancers(node string) (string, string, string, error) {
 	}
 	return lbTCP, lbUDP, lbSCTP, nil
 }
+
+// CreateLoadBalancerVIPs either creates or updates a set of load balancer VIPs mapping
+// from sourcePort on each IP of a given address family in sourceIPs, to targetPort on
+// each IP of the same address family in targetIPs
+func CreateLoadBalancerVIPs(lb string,
+	sourceIPs []string, sourcePort int32,
+	targetIPs []string, targetPort int32) error {
+	klog.V(5).Infof("Creating lb with %s, [%v], %d, [%v], %d", lb, sourceIPs, sourcePort, targetIPs, targetPort)
+
+	for _, sourceIP := range sourceIPs {
+		isIPv6 := utilnet.IsIPv6String(sourceIP)
+
+		var targets []string
```

Comment:
I use to initialise it, to avoid nil pointer deference errors , i.e. in this case UpdateLoadBalancer() may not be able to handle a nil if there are no target endpoints, it is safer
`targets := string{}`

Reply:
It's OK because UpdateLoadBalancer uses strings.Join on targets which is fine on a nil slice.